packages/ui/src/components/collectibles/collectibles.shared.tsx

```diff
@@ -0,0 +1,319 @@
+/* eslint-disable */
+
+export interface Ordinal {
+  id: string;
+  number: number;
+  output: string;
+  txid: string;
+  offset: string;
+  address: string;
+  preview: string;
+  title: string;
+  genesisBlockHeight: number;
+  genesisBlockHash: string;
+  genesisTimestamp: number;
+  value: string;
+  mimeType: string;
+  name: string;
+  src: string;
+}
+
+const mockOrdinals: Ordinal[] = [
```

Comment:
I obtained these mocks by logging all the inscriptions we have on Leather Dev wallet and picking out different samples. 

The structure here is what we are sending to the components but it seems different than the [BIS docs](https://docs.bestinslot.xyz/reference/api-reference/ordinals-and-brc-20-and-runes-and-bitmap-v3-api-mainnet+testnet+signet/inscriptions) and also what we have in `leather.io/query` for `[mock-inscriptions](https://github.com/leather-io/mono/blob/dev/packages/query/test/mock-inscriptions.ts)`

Then today I found this other `utils` that is mapping BIS to this format - [`createBestInSlotInscription`](https://github.com/leather-io/mono/blob/dev/packages/query/src/bitcoin/ordinals/inscription.utils.ts) 

I think it's OK for now and we can improve this and share the mocks better if needed / ditch these altogether when connecting the API. 

I found mocks